src/Sources/LightPortal/app.php

```diff
@@ -37,11 +40,29 @@
 	array_map($applyAlias, array_keys($aliases), $aliases);
 }
 
-// Development mode
+/**
+ * @bugo
+ * This will no longer be needed for development mode ;)
+ */
 if (is_file(__DIR__ . '/Libs/scssphp/scssphp/src/Compiler.php')) {
 	/** @noinspection PhpIgnoredClassAliasDeclaration */
 	class_alias('Bugo\\LightPortal\\Compilers\\Sass', 'Bugo\\LightPortal\\Compilers\\Zero');
 }
 
+// psr container
+$container = require 'config/container.php';
+
 // This is the way
-(new Integration())();
+$int = new Integration(
+	//$container->get(AddonHandler::class),
+	$container->get(Actions\Block::class),
+	$container->get(Actions\BoardIndex::class),
+	$container->get(Actions\Category::class),
+	$container->get(Actions\FrontPage::class),
+	$container->get(Actions\Page::class),
+	$container->get(Actions\Tag::class)
+);
+//$int->init();
+$int();
```

Comment:
Got the "Too few arguments to function Bugo\LightPortal\Integration::__construct(), 0 passed in \Sources\Subs.php on line 5978 and at least 1 expected" error